### PR TITLE
WIP: Hybrid 4.1.10 release

### PR DESCRIPTION
**Pull request by jogboms**

### CODE REVIEW

File: content/howto/mobile/include-local-bundled-resources.md [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+tags: ["mobile", "hybrid", "local", "ios", "apple"]
+---
+
+## 1 Introduction
+
+If you have ever wanted to include a local copy of your deployed cloud resources into your distributed iOS app, then, this how-to is tailored just for you.
+
+## 2 Prerequisites
+
+To begin this process, you will need to build versions versions of your mobile application:
```
Comment (mohannaresh) [inline content/howto/mobile/include-local-bundled-resources.md]: versions word repeated for twice.

**Comment by ConnorLand** [comment]
Hi @jogboms . I see that this PR has 2 portions: updates to the Hybrid App Base, and a new document. I'll take the Hybrid App Base changes out of this PR and commit them now myself. Then I will get to the new document portion of this PR later. 

If this comes up again, could you please send 2 separate PRs so I can process the quick one easily? Thank you for contributing to the documentation Jeremiah.

**Comment by ConnorLand** [comment]
I have published the 2 bullets for the Jan 3 release notes. When @smarki returns, he and I can discuss the forthcoming update and its document (which pertain to local bundled resources). I want to make sure this new update and doc are necessary, and not too narrow in use.

**Comment by jogboms** [comment]
Thanks @ConnorLand 

I also just repurposed the PR as this now points to v4.1.10 and would be released soon

**Comment by jogboms** [comment]
Is is okay i create a new PR and close this?

**Comment by ConnorLand** [comment]
> Is it okay to create a new PR and close this?

Sure @jogboms . I can close this one.

**Comment by ConnorLand** [comment]
Closing this so a newer, cleaner one can be submitted.

**Pull request closed without merging**